content/applications/studio/automated_actions.rst

```diff
@@ -142,44 +138,87 @@ Timing Conditions
 Trigger automated actions at a point in time relative to a date field or to the creation or update
 of a record. The following triggers are available:
 
-- :guilabel:`Based on date field`: The action is triggered a defined period of time before or after
-  the date of the selected date field.
+- :guilabel:`Based on date field`: The action is triggered a defined period of time *before or
+  after* the date of the selected date field, e.g., 30 days before the contract end date.
 - :guilabel:`After creation`: The action is triggered a defined period of time after a record is
   created and saved.
 - :guilabel:`After last update`: The action is triggered a defined period of time after an existing
   record is edited and saved.
 
-You can then define:
+To configure a :guilabel:`Timing Conditions` trigger, with :ref:`developer mode activated
+<developer-mode>`:
+
+#. Select the appropriate :guilabel:`Trigger` from the :guilabel:`Timing Conditions` category.
+   If you select the :guilabel:`Based on date field` trigger, select the relevant date field.
+#. Configure the :guilabel:`Delay` to determine the *execution date time* of the action(s). To
+   do so, enter the number of :guilabel:`Minutes`, :guilabel:`Hours`, :guilabel:`Days`, or
+   :guilabel:`Months` after which the action should be triggered. If you selected the
+   :guilabel:`Based on date field` trigger, the action can be triggered :guilabel:`After` or
+   :guilabel:`Before` the selected date field.
+#. Optionally, :ref:`add conditions <studio/automated-actions/conditions>` that the record must meet
+   before and/or after the rule is triggered.
+#. Click :icon:`fa-cloud-upload` :guilabel:`Save manually`.
+#. Click :icon:`fa-arrow-right` :guilabel:`Scheduled action` to open the :guilabel:`Automation
+   Rules: check and execute` scheduled action.
+
+   .. tip::
+      By default, this scheduled action is set to :guilabel:`Execute Every` 4 hours, i.e., to check
```

Comment:
I'm trying on a 19.0 trial, and the default is 1 minute (on a 19.0 runbot, it's 6 minutes, and on 18.0 it's 4 hours)
<img width="580" height="270" alt="image" src="https://github.com/user-attachments/assets/804afe40-ef9f-40df-a1fc-9a549a423b43" />